example/path/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
`path` can set when access to url under path value.
ex) when access to `http://domain/path/`,  can set `path=/path`. if not, will ignore set cookie.